pages/cadastro/index.public.js

```diff
@@ -188,12 +191,14 @@ function SignUpForm() {
           setErrorObject={setErrorObject}
         />
 
-        <Box sx={{ display: 'flex', alignItems: 'center', gap: 3 }}>
-          <Checkbox checked={isTermsAccepted} onClick={() => setIsTermsAccepted(!isTermsAccepted)} />
-          <Text>
-            Li e estou de acordo com os
-            <Link href="/termos-de-uso"> Termos de Uso.</Link>
-          </Text>
+        <Box>
+          <FormControl>
+            <Checkbox checked={isTermsAccepted} onClick={() => setIsTermsAccepted(!isTermsAccepted)} />
```

Comment:
Para adequar ao que temos na documentação:

```suggestion
            <Checkbox checked={isTermsAccepted} onChange={() => setIsTermsAccepted(!isTermsAccepted)} />
```